admin/includes/template/templates/tplIndexStartWizard.php

```diff
@@ -94,33 +94,11 @@
    </fieldset>
    </form>
 <script>
-   $().ready(function() {
-       $.validator.messages.required = '<?php echo TEXT_FORM_ERROR_REQUIRED; ?>';
-       $("#setup-wizard").validate({
-         errorElement: 'span',
-         errorClass: 'help-inline invalid',
-         submitHandler: function(form) {
-          form.submit();
-         },
-         rules: {
-           store_name: "required",
-           store_owner: "required",
-           store_owner_email: "required email",
-           store_zone: {
-              required: true,
-              min: 0
-            }
-        },
-         messages: {
-            store_zone: '<?php echo TEXT_FORM_ERROR_CHOOSE_ZONE; ?>'
-         }
-       });
-     });
    $(function()
        {
      $('#store_country').change(function(e) {
       zcJS.ajax({
-        url: "zcAjaxHandler.php?act=installWizard&method=getZones",
+        url: "<?php echo zen_href_link($_GET['cmd'], zen_get_all_get_params(array('action')) . "action=getZones"); ?>",
```

Comment:
Do we need `zen_get_all_get_params()` here? Not really a problem, but on the installWizard screen there are no other `GET` params that would be needed/relevant.

I suppose if the wizard-related settings got unset such that the wizard was re-triggered then the cameFrom param might be valuable ... but I still  think it's moot.

Again, not a big deal, but less is more.